docs/robotpy_ext.control.rst

```diff
@@ -8,6 +8,13 @@ robotpy_ext.control.button_debouncer module
     :members:
     :undoc-members:
 
+robotpy_ext.control.toggle module
+---------------------------------
+
+.. automodule:: robotpy_ext.control.toggle
+    :members:
+    :undoc-members:
+
```

Comment:
Could we please have a consistent number of blank lines between sections here? IMHO 1 blank line is enough.